sparkviz/__init__.py

```diff
@@ -18,6 +18,9 @@
 import pyspark.sql as sd
 from pyspark.storagelevel import StorageLevel
 import pandas as pd
+# Turn on pandas application/vnd.dataresource+json for use with
+# https://github.com/nteract/nteract/issues/1526
+pd.options.display.html.table_schema = True
```

Comment:
There is a part of me that feels bad sneaking in side effects that push the direction I want. 😄

Reply:
Legit, I'll revert it out when we do this with the metadata instead of just using pandas.

Reply:
I though there was a way to coerce pandas to give you this representation, but it appears that `df._repr_data_resource_()` is a noop if `table_schema` isn't set.